frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -278,7 +277,7 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
         const scList = _.filter(storageClasses, (sc) => sc.provisioner === provisioner);
         // take the default storageclass
         _.forEach(scList, (sc) => {
-          if (sc.metadata && _.isEqual(sc.metadata.annotations, defaultSAnotations)) {
+          if (isDefaultClass(sc)) {
```

Comment:
What if the Provisioner storage class is not Default? I think then we should select one of one or many storage class present.

Reply:
Here, we are selecting default storage class for that provisioner. Didn't get you. Can you please elaborate?

Reply:
OF what if the aws based storage class does not set to default. what if someone has removed the annotations of default class then also we should be able to select the aws storage class.

first priority should be given to default storage class but if that's not available we should check for normal storage class which is backed by a provisioner.

Reply:
@cloudbehl This PR is just for handling the follow up changes from the previous PR. Storage class handling part in case,its not default can be handled in a separate PR. Can you please raise an issue for that.

Reply:
Ack I will create a separate issue.